SPEC.md

```diff
@@ -160,7 +160,7 @@ Network configuration in JSON format must be streamed to the plugin through stdi
 
 Note that IPAM plugins should return an abbreviated `Result` structure as described in [IP Allocation](#ip-allocation).
 
-Plugins must indicate success with a return code of zero and the following JSON printed to stdout in the case of the ADD command. The `ips` and `dns` items should be the same output as was returned by the IPAM plugin (see [IP Allocation](#ip-allocation) for details) except that the plugin should fill in the `interface` indexes appropriately, which are missing from IPAM plugin output since IPAM plugins should be unaware of interfaces.
+Plugins must indicate success with a return code of zero and the following JSON printed to stdout in the case of the ADD command. The `ips`, `dns` and `capabilityArgs` items should be the same output as was returned by the IPAM plugin (see [IP Allocation](#ip-allocation) for details) except that the plugin should fill in the `interface` indexes appropriately, which are missing from IPAM plugin output since IPAM plugins should be unaware of interfaces.
```

Comment:
I found it surprising that `capabilityArgs` would come specifically from the IPAM plugin.  Is the background to this described somewhere?

Reply:
TBH, I'm not sure about this, I just see [this comment](https://github.com/containernetworking/cni/issues/582#issuecomment-537572565). Personally, I'd prefer `Args`. The original intention about adding this optional args is that I want to consume the extra args returned by IPAM plugin result, for now, [it](https://github.com/containernetworking/cni/blob/master/pkg/types/current/types.go#L125) doesn't support carry extra args.

Reply:
I don't see where either of your links mention IPAM.

Reply:
So what's your suggestion? How about make this field not specifically to ipam plugin? I think this field is also useful for those chained plugins.

Reply:
Yes, I cannot see any reason to make this specific to IPAM, so I would change the wording so it doesn't say that.